src/posix/platform/CMakeLists.txt

```
@@ -38,15 +38,15 @@ if(OT_POSIX_VIRTUAL_TIME)
     list(APPEND OT_PRIVATE_DEFINES "OPENTHREAD_POSIX_VIRTUAL_TIME=1")
 endif()
 
+set(OT_POSIX_CONFIG_RCP_BUS "UART" CACHE STRING "RCP bus type")
+list(APPEND OT_PLATFORM_DEFINES "OPENTHREAD_POSIX_CONFIG_RCP_BUS=OT_POSIX_RCP_BUS_${OT_POSIX_CONFIG_RCP_BUS}")
```

according to suggestion from @abtink, we should make default value `""`, and only set when the variable is set `""`.
```suggestion
set(OT_POSIX_CONFIG_RCP_BUS "" CACHE STRING "RCP bus type")
if(OT_POSIX_CONFIG_RCP_BUS)
    list(APPEND OT_PLATFORM_DEFINES "OPENTHREAD_POSIX_CONFIG_RCP_BUS=OT_POSIX_RCP_BUS_${OT_POSIX_CONFIG_RCP_BUS}")
endif()
```